### PR TITLE
Allow one player to click on a free cell and add marble

### PR DESCRIPTION
**Pull request by Clebiez**
# (0.5) As Brice, I want to place marbles on a board

- [x] Create `BoardService` to handle logic for board.
- [x] Create `GameController` to handle basic actions for a player.
- [x] Allow user to create a new game by visiting `/` route 
- [x] By clicking on a cell, change the value to 1. 
- [x] Game state is actually stored in a cookie, by reloading the browser you can retrieve same board. 

sf-addmarble

### CODE REVIEW

File: public/css/game.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.cell {
+    width: 70px;
+    height: 70px;
+    flex: 1;
+    border: none; /* Resetting default browser borders */
+    border-right: 1px solid;
+    border-bottom: 1px solid;
+    border-color: var(--main-bg-color);
```
Comment (zyhou) [inline public/css/game.css:23]: css var <3

File: templates/game/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{% block body %}
+
+<form action="{{action}}" method="POST" class="game-container">
+    {% for row in board %}
+        <div class="row">
+        {% for value in row %}
+            <button
+                type="submit"
+                name="position"
+                value="{{loop.parent.loop.index}}-{{loop.index}}"
```
Comment (zyhou) [inline templates/game/index.html.twig:26]: How this work ? Sometimes PHP loses me

File: src/Service/BoardService.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+    class BoardService
+    {
+        public const BOARD_LENGTH = 6;
+
+        public function initBoard(): array
+        {
+            // Return a 6*6 2 dimensions array filled with 0 as value for each cell.
+            return array_fill(0, 6, array_fill(0, 6, 0));
```
Comment (zyhou) [inline src/Service/BoardService.php]: use constant in this function/file (`initBoard`, `isOutsideBoard`)

File: src/Service/BoardService.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            if ($position[0] < 0 || $position[0] >= 6 || $position[1] < 0 || $position[1] >= 6) {
+                return true;
+            }
+
+            return false;
+        }
+
+        public function isPositionAvailable($board, $position): bool
+        {
+            if($this->isOutsideBoard($position) == true) {
```
Comment (zyhou) [inline src/Service/BoardService.php]: no need to add `==true`. PHP is different between one or two egal (1 == true is true but 1 === true is false

File: src/Service/BoardService.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            }
+
+            return true;
+
+        }
+
+        public function addMarble($board, $position): array
+        {
+           
+
```
Comment (zyhou) [inline src/Service/BoardService.php]: Becare about empy space. Maybe you can add some linter

**Review by zyhou — CHANGES_REQUESTED** [review]

File: public/css/game.css (hunk 1 above)
Comment (Clebiez) [reply public/css/game.css:23]: Less is useless :D

**Review by zyhou — APPROVED** [review]

**Pull request merged**